### PR TITLE
chore: use github actions

### PR DESCRIPTION
**Pull request by fallenoak**

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node_version: [10.x, 12.x, 14.x]
```
Comment (timkurvers) [inline .github/workflows/pull_request.yml:11]: 👍

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Use Node.js ${{ matrix.node_version }}
+        uses: actions/setup-node@v1
+        with:
+          node_version: ${{ matrix.node_version }}
+
+      - run: npm install
+
+      - run: npm test
+        env:
+          CI: true
```
Comment (timkurvers) [inline .github/workflows/pull_request.yml:25]: Do we perhaps want to run `npm run lint` too? (not sure whether this repo has it)

Comment (fallenoak) [reply .github/workflows/pull_request.yml:25]: `test` runs `lint` (which I generally think we should be doing):
```json
"test": "npm run lint && jest --config .jestrc.json"
```

Comment (timkurvers) [reply .github/workflows/pull_request.yml:25]: Ah! 👍

**Pull request merged**